### ISSUE TITLE
Nodes directory is empty

### ISSUE THREAD

**Issue by jquacinella** [sue445/jenkins-backup-script#37]
Build fails since nodes/ dir is empty:

Output:
```
+ '[' -d /var/jenkins_home/nodes/ ']'
+ cp -R '/var/jenkins_home/nodes/*' '/var/jenkins_home/jobs/Backups - Daily Job/workspace/tmp/jenkins-backup/nodes'
cp: cannot stat ‘/var/jenkins_home/nodes/*’: No such file or directory
Build step 'Execute shell' marked build as failure
[ExclusiveBuildWrapper] Canceling Jenkins shutdown mode... 
Finished: FAILURE
```

You could use an extra conditional to prevent this from happening if dir is empty (http://www.cyberciti.biz/faq/linux-unix-shell-check-if-directory-empty/), like:

`if [ "$(ls -A $DIR)" ]; then`

**Comment by sue445** [comment]
@jquacinella thank you! 
I fixed and released v0.1.4

**Comment by jquacinella** [comment]
Thanks for the quick response, you rock!
